docs/upgrade-guides/1.x-to-2.0.0.md

```diff
@@ -0,0 +1,74 @@
+
+### New XML namespaces
+
+New XML namespaces have been defined, the following is the list of the new namespaces:
+
+- `http://schemas.radicalframework.com/windows/behaviors`
+- `http://schemas.radicalframework.com/windows/input`
+- `http://schemas.radicalframework.com/windows`
+- `http://schemas.radicalframework.com/windows/converters`
+- `http://schemas.radicalframework.com/windows/controls`
+- `http://schemas.radicalframework.com/windows/effects`
+- `http://schemas.radicalframework.com/windows/markup`
+- `http://schemas.radicalframework.com/windows/regions` 
+
+The existing XML namespaces, starting with `http://schemas.topics.it/wpf/radical/*`, can still be used but should be considered as obsolete and will be removed in V3.
+
+### Breaking changes
+
+#### Message broker `IMessage` support
+
+The `IMessage` interface has been removed, message broker supports only POCO messages in V2.
+
+#### `IEntityView` custom properties
+
+Entity view `AddPropertyMapping` has been renamed to `AddCustomProperty`, `RemovePropertyMapping` to `RemoveCustomProperty`, and `IsPropertyMappingDefined` to `IsCustomPropertyDefined`
+
+#### XML namespaces
+
+- XML namespace definition `http://schemas.topics.it/wpf/radical/windows/presentation/regions` changed to `http://schemas.topics.it/wpf/radical/windows/regions`.
+- XML namespace definition `http://schemas.topics.it/wpf/radical/windows/presentation/behaviors` changed to `http://schemas.topics.it/wpf/radical/windows/behaviors`.
+
+#### Removed and changed types and namespaces
+
+- The type `Fact` has been removed. [`AutoCommandBinding`](https://docs.radicalframework.com/markup-extensions/auto-command-binding) supports boolean properties and regular property changed notifications.
+- The `RadicalWpfApplication` type has been removed. If a `DoEvents`-like implementation is really needed original code can be copy/pasted from [GitHub commits history](https://github.com/RadicalFx/Radical.Windows/blob/fa52500164e26f351055034ffbcb87a085b64e4b/src/Radical.Windows/RadicalWpfApplication.cs#L6-L42).
+- `EntityValidationService<T>` has been removed in favor of `DataAnnotationValidationService<T>`.
+- `ApplicationBootstrapper` has been obsoleted in favor of the new `AddRadicalApplication<TMainView>()` extension method.
+  - All virtual members are now sealed.
+  - It will be removed in v3.
+- `GetValidationService()` has been removed from `AbstractViewModel` and `AbstractMementoViewModel`. Use the new protected `ValidationService` property to set the current validation service instance.
+- `Radical.Windows.Presentation` namespace has been merged into `Radical.Windows`.
+- the `Topics` namespace prefix has been dropped, the new root namesapce is `Radical`.
+- `WpfDispatcher` type has been made internal.
+
+#### Behavior changes
+
+- External IoC containers are not supported. Support for non default IoC containers will be available in a future minor release
```

Comment:
considering workitem "third-party containers (# 113)" this content should be updated